RecoTracker/LSTCore/src/alpaka/Triplet.h

```diff
@@ -503,17 +503,15 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
         alpaka::syncBlockThreads(acc);
 
         // Step 1: Make inner and outer SG pairs
-        for (unsigned int innerSegmentArrayIndex = threadIdY; innerSegmentArrayIndex < nInnerSegments;
-             innerSegmentArrayIndex += blockSizeY) {
+        for (unsigned int outerSegmentArrayIndex : cms::alpakatools::uniform_elements_y(acc, nInnerSegments)) {
           unsigned int innerSegmentIndex =
-              ranges.segmentRanges()[innerInnerLowerModuleIndex][0] + innerSegmentArrayIndex;
+              ranges.segmentRanges()[innerInnerLowerModuleIndex][0] + innerTripletArrayIndex;
```

Comment:
Just checking, were these changes (`innerSegmentArrayIndex` -> `outerSegmentArrayIndex`, `innerSegmentArrayIndex` -> `innerTripletArrayIndex`) intended ?

Reply:
Sorry, this is a problem introduced when doing the change. I am fixing it and rerun validation plots.

Reply:
The standalone validation has been performed, physics performance remains in good agreement